src/Pages/MainPage/MainPage.js

```diff
@@ -1,33 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState } from './node_modules/react';
 //style files  for mdbreact
-import '@fortawesome/fontawesome-free/css/all.min.css';
-import 'bootstrap-css-only/css/bootstrap.min.css';
-import 'mdbreact/dist/css/mdb.css';
+import './@fortawesome/fontawesome-free/css/all.min.css';
+import './bootstrap-css-only/css/bootstrap.min.css';
+import './mdbreact/dist/css/mdb.css';
```

Comment:
you are using a lot of css files and this will make the website so slow
try to remove the unused ones or if you use only a small piece of it try to move it to tailwind
also you have an error here when I want to run the app

Reply:
I think this is because the `./` at the beginning 

```suggestion
import '@fortawesome/fontawesome-free/css/all.min.css';
import 'bootstrap-css-only/css/bootstrap.min.css';
import 'mdbreact/dist/css/mdb.css';
```